main.tex

```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
 develop, and contribute to those tools within a conscientious and welcoming
 community.
 
-\acknowledgments
-
-Who to thank?  numfocus, GSOC, Python Software Foundation
-
-We thank the many web services that make developing and maintaining this package
-feasible, such as GitHub, Travis CI, Appveyor, CircleCI, and Read the Docs.
+\acknowledgments The \astropy community is supported by and makes use
+of a number of organizations and services outside the traditional
+academic community. We thank Google for financing and organizing the
+Google Summer of Code (GSoC) program, that has funded severals
+students per year to work on \astropy related projects over the
+summer. These students ofter turn into long-term contributors. We also
+thank NumFOCUS and the Python Software Foundation for financial
+support for various needs. Within the academic community, we thank
+institutions that make it possible that astronomers and programmers on
+their staff can contribute (part of) their time to the development of
+\astropy projects. Furthermore, the \astropy packages would not exisit
+in their current form without a number of webservices for code
+hosting, continuous integration, and documentation; in particular,
+\astropy heavily relies on GitHub, Travis CI, Appveyor, CircleCI, and
```

Comment:
Perhaps add links to the websites here if they are not linked or cited elsewhere?